ui/src/app/pages/datasets/add/dataset.add.component.html

```diff
@@ -91,6 +91,14 @@ <h4>{{isEdit ? 'Edit Dataset' : 'New Dataset'}}</h4>
             </nb-form-field>
             <hr/>
             <div class="d-flex justify-content-end">
+              <button
+                (click)="goBack()"
+                data-orb-qa-id="cancel"
+                ghost
+                nbButton
+                status="primary"
+                type="button">Cancel
```

Comment:
@gpazuch it would be good to follow the same approach of the agent group add component. May we should create an object to hold those label information? It will allow us to change all affected pages at once by only changing on file.

What do you think about it?

Reply:
The same applies to the other files of this PR.

Reply:
Sure thing. There is a place for that.

Reply:
Would you address this fix on this PR, or should I open an issue to it?

Reply:
I'm fixing it right away

Reply:
Fixed